logic/mdwLogic.js

```diff
@@ -0,0 +1,59 @@
+const axios = require('axios');
+const logger = require('../utils/logger')(module);
+
+const LIMIT = 100; // max 1000
+const knownHashes = {};
+
+module.exports = class MdwLogic {
+  // fetches pages forwards, if no next its the last page, don't cache that
+  static async iterateMdw(next, abortIfHashKnown = false) {
+    const url = `${process.env.MIDDLEWARE_URL}/${next}`;
```

Comment:
no default url?

Reply:
not needed probably, what do you think?

Reply:
Lets leave it as is and at some point we can move everything to a proper config file with defaults.